CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-## [3.1.1] - TBD
+## [4.0.0] - TBD
+### Added
+- Adding `beekeeper-api` module.
```

Comment:
What about putting Added? I never saw the -ing verb here. (Are you going to put the release date?)

Reply:
I put the release date in another PR